pages/common/aws-lambda.md

```diff
@@ -0,0 +1,32 @@
+# aws lambda
+
+> CLI for AWS lambda.
+> More information: <https://docs.aws.amazon.com/cli/latest/reference/lambda/index.html>.
+
+- Run a function:
+
+`aws lambda invoke --function-name {{function_name}} {{local_file_to_store_results}}`
+
+- Run a function with payload as input:
+
+`aws lambda invoke --function-name {{function_name}} --payload {{payload_in_json}} {{local_file_to_store_results}}`
+
+- List functions:
+
+`aws lambda list-functions`
+
+- Get function configuration:
+
+`aws lambda get-function-configuration --function-name  {{function_name}}`
```

Comment:
`function_name` is redundant because the argument said `function-name`. So:
```suggestion
`aws lambda get-function-configuration --function-name  {{name}}`
```